addons/im_livechat/models/mail_channel.py

```diff
@@ -138,6 +138,10 @@ def _define_command_history(self):
             'help': _('See 15 last visited pages')
         }
 
+    def _execute_command_help_message(self):
+        msg = super(MailChannel, self)._execute_command_help_message()
+        return msg + _("Type <b>:shortcut</b> to insert a canned response in your message.<br>")
```

Comment:
In theory, not good to concatenate translatable strings, because format of string usually depends on the language. Here it's rather safe.